### ISSUE TITLE
Disable showing of stacktraces in output window

### ISSUE THREAD

**Issue by andlehma** [BetterThanTomorrow/calva#819]
I am very new to Clojure and Calva, so forgive me if this is trivial. When I am using the REPL, and I make some mistake which gives an error, I get something like the following:

```
clj::test.core=>
("hello")
; Execution error (ClassCastException) at test.core/eval15567 (form-init10260932038297751257.clj:134).
; class java.lang.String cannot be cast to class clojure.lang.IFn (java.lang.String is in module java.base of loader 'bootstrap'; clojure.lang.IFn is in unnamed module of loader 'app')
[{:file "file:/home/andrew/src/test/src/test/core.clj:134" :fn "eval15567"  :method "invokeStatic" :flags [:project :repl :clj]}
 {:file "file:/home/andrew/src/test/src/test/core.clj:134" :fn "eval15567"  :method "invoke" :flags [:dup :project :repl :clj]}
 {:file "Compiler.java" :line 7177 :method "eval" :flags [:tooling :java]}
 {:file "Compiler.java" :line 7132 :method "eval" :flags [:dup :tooling :java]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/core.clj:3214" :fn "eval"  :method "invokeStatic" :flags [:clj]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/core.clj:3210" :fn "eval"  :method "invoke" :flags [:clj]}
 {:file "jar:file:/home/andrew/.m2/repository/nrepl/nrepl/0.8.2/nrepl-0.8.2.jar!/nrepl/middleware/interruptible_eval.clj:87" :fn "evaluate/fn/fn"  :method "invoke" :flags [:tooling :clj]}
 {:file "AFn.java" :line 152 :method "applyToHelper" :flags [:java]}
 {:file "AFn.java" :line 144 :method "applyTo" :flags [:java]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/core.clj:665" :fn "apply"  :method "invokeStatic" :flags [:clj]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/core.clj:1973" :fn "with-bindings*"  :method "invokeStatic" :flags [:clj]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/core.clj:1973" :fn "with-bindings*"  :method "doInvoke" :flags [:dup :clj]}
 {:file "RestFn.java" :line 425 :method "invoke" :flags [:java]}
 {:file "jar:file:/home/andrew/.m2/repository/nrepl/nrepl/0.8.2/nrepl-0.8.2.jar!/nrepl/middleware/interruptible_eval.clj:87" :fn "evaluate/fn"  :method "invoke" :flags [:tooling :clj]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/main.clj:437" :fn "repl/read-eval-print/fn"  :method "invoke" :flags [:clj]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/main.clj:437" :fn "repl/read-eval-print"  :method "invoke" :flags [:dup :clj]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/main.clj:458" :fn "repl/fn"  :method "invoke" :flags [:clj]}
 {:file "jar:file:/home/andrew/.m2/repository/org/clojure/clojure/1.10.1/clojure-1.10.1.jar!/clojure/main.clj:458" :fn "repl"  :method "invokeStatic" :flags [:dup :clj]}
 {:file "jar:file:/home/andrew/.m2/repository/nrepl/nrepl/0.8.2/nrepl-0.8.2.jar!/nrepl/middleware/interruptible_eval.clj:115" :fn "evaluate"  :method "invokeStatic" :flags [:tooling :clj]}
 {:file "jar:file:/home/andrew/.m2/repository/nrepl/nrepl/0.8.2/nrepl-0.8.2.jar!/nrepl/middleware/interruptible_eval.clj:150" :fn "interruptible-eval/fn/fn"  :method "invoke" :flags [:tooling :clj]}
 {:file "AFn.java" :line 22 :method "run" :flags [:java]}
 {:file "jar:file:/home/andrew/.m2/repository/nrepl/nrepl/0.8.2/nrepl-0.8.2.jar!/nrepl/middleware/session.clj:202" :fn "session-exec/main-loop/fn"  :method "invoke" :flags [:tooling :clj]}
 {:file "jar:file:/home/andrew/.m2/repository/nrepl/nrepl/0.8.2/nrepl-0.8.2.jar!/nrepl/middleware/session.clj:201" :fn "session-exec/main-loop"  :method "invoke" :flags [:tooling :clj]}
 {:file "AFn.java" :line 22 :method "run" :flags [:java]}
 {:file "Thread.java" :line 832 :method "run" :flags [:java]}]

```

The first two lines are the useful error, then there are a bunch of lines of detailed Java errors which are irrelevant to me. I would like to disable them and just see the Clojure errors if possible.

**Comment by andlehma** [comment]
I am running on Arch Linux.

```
$ java --version                                              
openjdk 14.0.2 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12)
OpenJDK 64-Bit Server VM (build 14.0.2+12, mixed mode)
```

**Comment by PEZ** [comment]
Hello. There is no way currently. Improving error output is in the works. Chip in on this issue: #737

**Comment by bpringe** [comment]
I've closed #737 since we released conciser stacktraces and renamed this to be specific to disabling the showing of stacktraces.

**Comment by bpringe** [comment]
I suppose we should add a setting for enabling the printing of stack traces which will default to true.
